NTO/Automation/entities/Configuration.ttl

```diff
@@ -26,12 +26,13 @@ ogit.Automation:Configuration
 
 	);
 	ogit:optional-attributes (
-		
+
 	);
 	ogit:indexed-attributes (
-		
+
 	);
 	ogit:allowed (
+            [ ogit:uses  ogit.Automation:ActionHandler ]
 			[ ogit:uses  ogit.Automation:KIRanking ]
```

Comment:
shouldn't these be on the same indent level?

Reply:
There were spaces instead of tabs, fixed.